GVFS/GVFS.FunctionalTests/Tests/GitCommands/StatusTests.cs

```diff
@@ -25,6 +25,19 @@ public void MoveFileIntoDotGitDirectory()
             this.ValidateGitCommand("status");
         }
 
+        [TestCase]
+        public void DeleteThenCreateThenDeleteFile()
+        {
+            string srcPath = @"Readme.md";

```

Comment:
Should we use a different path here to avoid impacting the `MoveFileIntoDotGitDirectory` test?

Reply:
These tests are set to an enlistment per tests so it should never impact other tests right?

Reply:
base(enlistmentPerTest: true

I would think that's fine actually

Reply:
@kewillford  Ahh you're right, sorry about that!